projects/packages/videopress/src/client/admin/components/video-storage-meter/stories/style.module.scss

```diff
@@ -0,0 +1,3 @@
+.wrapper {
+  width: 358px;
```

Comment:
I see we are defining width in the stories for testing purposes. I'm not sure about it, considering that it has the screen-size addons:


https://user-images.githubusercontent.com/77539/187513880-06f127c8-cfc6-46fc-8b86-7af927a34375.mov

Reply:
Yes. `VideoUploadArea` also does that. I will remove it on both.

Reply:
yeah, I knew you knew it